.gitignore

```diff
@@ -72,4 +73,7 @@ target/
 .python-version
 
 # pytest
-.pytest_cache/
\ No newline at end of file
+.pytest_cache/
+
+# IntelliJ
+.idea
```

Comment:
Tip: having a global gitignore can avoid having to add this to new repos.

Reply:
True, want me to remove this?

Reply:
No need :) you can leave it there